README.md

```diff
@@ -17,12 +17,17 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
 ## Java
 - [Catima](https://github.com/TheLastProject/Catima) -  Catima, a Loyalty Card & Ticket Manager for Android
 - [F-Droid](https://gitlab.com/fdroid/fdroidclient) - F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform. The client makes it easy to browse, install, and keep track of updates on your device.
+- [Slimefun](https://github.com/Slimefun/Slimefun4)  - Turn your Minecraft Spigot server into a modpack with over 500+ items added! **Tree planted for every approved Hacktoberfest PR**
 
 ## Kotlin
 - [Valetudo Companion](https://github.com/Hypfer/valetudo-companion) -  Valetudo Companion App for Android
 
 ## Python
 - [F-Droid Server](https://gitlab.com/fdroid/fdroidserver/) - Tools for managing builds, indexes, updates, and deployments for F-Droid repositories.
 
+## JavaScript
+- [Cloudflare Status Worker](https://github.com/WalshyDev/cf-status-worker) - Get a message in your Discord server when Cloudflare has an issue. Deployed with Clouflare Workers
```

Comment:
Please add some GitHub issues to this repo so people know what to contribute to

Reply:
Thought I'd added these haha, done :)